src/main/java/pages/HomePageFS.java

```diff
@@ -0,0 +1,22 @@
+package pages;
+
+import chunks.GalleryOverlay;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePageFS {
+
+    @FindBy(xpath = "//div[@class='Gallery Hero-gallery']/button[@class='Button--underlined--dark Gallery-button']/span[@class='Button-label']")
+    private WebElement gallery_link;
+
+
+    //Methods to interact on all page's elements
```

Comment:
Switch with Element-getters. Is easier to find the methods/interactions first and then the element definition

Reply:
Done